test/acceptance/features/bindToService.feature

```diff
@@ -216,3 +216,40 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
+
+    Scenario: Bind an imported quarkus app which is deployed as knative service to PostgreSQL database
```

Comment:
For an easier user/human navigation, I would extract this scenario into a separate feature file, called `bindKnativeService.feature`

Reply:
Agreed, but please make a note that we have to add steps under 'Background' to this feature file as well.

Reply:
yes, that is fine.